wiki/Gameplay/Judgement/osu!mania/en.md

```diff
@@ -1,35 +1,68 @@
+| PERFECT | 16 |
+| GREAT | 34 if OD > 4, otherwise 47 |
+| GOOD | 67 if OD > 4, otherwise 77 |
+| OK | 97 |
+| MEH | 121 |
+| MISS | 158 |
 
-| Image | Name | Effect |
-| :-: | :-: | :-- |
-| ![](/wiki/shared/judgement/osu!mania/mania-hit300g.gif "Rainbow 300") | Rainbow 300 (MAX) | Perfect hit of the note or hold. Comparing to 300, getting this mark required dead-on accuracy to get it. Otherwise, a regular 300 will be given. Completing the beatmap with just this score will give the maximum 1,000,000 score (1 million score limit). This is equivalent to *Marvelous* in DDR term. |
-| ![](/wiki/shared/judgement/osu!mania/mania-hit300.png "300") | 300 | Almost perfect hit of the note or hold. Maximum score limit cannot be achieved if this score was given. |
-| ![](/wiki/shared/judgement/osu!mania/mania-hit200.png "200") | 200 | Slightly early or slightly late hit or hold. New players may get this mark a lot until they got accustomed to the mechanism of the conveyor, speed, buttons and the judgement area. |
-| ![](/wiki/shared/judgement/osu!mania/mania-hit100.png "100") | 100 | Early or late hit or hold. It does not affect the HP bar at all. |
-| ![](/wiki/shared/judgement/osu!mania/mania-hit50.png "50") | 50 | Very early or very late hit or hold. This score will be given for unreleased hold notes by default. Decreases HP, but not for a drastic amount. Does not break combo. |
-| ![](/wiki/shared/judgement/osu!mania/mania-hit0.png "Miss") | Miss | Not hitting the note or holding the note using the correct keys. Drastically drain the HP and breaks the current combo. |
+Rate-changing mods ([Double Time](/wiki/Gameplay/Game_modifier/Double_Time), [Half Time](/wiki/Gameplay/Game_modifier/Half_Time), and [Nightcore](/wiki/Gameplay/Game_modifier/Nightcore)) do not affect hit window durations in osu!mania.<!-- unique to osu!mania, not a thing in lazer. internal reference: https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/GameplayElements/HitObjectManagerMania.cs#L151-L160 -->
 
 ## Judgement mechanics
 
-**Notes:**
+### Notes
+
+A note is judged with a PERFECT, GREAT, GOOD, OK, MEH, or MISS depending on how accurately it is hit. Hitting a note before the MISS window has no effect, and not hitting a note will cause a miss after the OK window passes (late MEH hits are impossible).
+
+### Hold notes
+
+<!-- internal reference: https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/GameplayElements/HitObjects/Mania/HitCircleManiaLong.cs#L235-L296 -->
+
+Hold notes are given one judgement depending on the timing of both the keypress at the head and the release at the tail, according to the following table, where the *combined hit error* is *head hit error* + *tail release error* (both being positive):
+
+| Judgement | Requirement |
+| :-: | :-- |
+| PERFECT | Head hit error <= max error for PERFECT \* 1.2 **AND** combined hit error <= max error for PERFECT \* 2.4 |
```

Comment:
may I suggest using the multiplication sign, `×`, instead, to avoid excessive escaping?

Reply:
put in for ≤, ≥ and ×